c2rust-transpile/src/lib.rs

```diff
@@ -88,10 +88,32 @@ pub struct TranspilerConfig {
 }
 
 impl TranspilerConfig {
-    fn is_binary(&self, file: &Path) -> bool {
+    fn binary_name_from_path(file: &Path) -> String {
         let file = Path::new(file.file_stem().unwrap());
-        let name = get_module_name(file, false, false, false).unwrap();
-        self.binaries.contains(&name)
+        get_module_name(file, false, false, false).unwrap()
+    }
+
+    fn is_binary(&self, file: &Path) -> bool {
+        let module_name = Self::binary_name_from_path(file);
+        self.binaries.contains(&module_name)
+    }
+
+    fn check_binaries(&self, modules: impl IntoIterator<Item = impl AsRef<Path>>) -> bool {
```

Comment:
```suggestion
    fn check_all_binaries_used(&self, modules_to_transpile: impl IntoIterator<Item = impl AsRef<Path>>) -> bool {
```

I think these names make it clear what the function will do without reading its body.

Reply:
I agree, though I think `check_if_all_binaries_used` is a bit clearer still.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/614/commits/fe5c78369af8d3e04ae242b15c9fabfe54fb15e7.